.github/workflows/docker.yml

```diff
@@ -15,10 +15,21 @@ jobs:
       - name: Get current date
         id: date
         run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+      - name: Set up QEMU
```

Comment:
Why?

Reply:
Im not sure if we can remove it. The new docker actions use these and in the docs it always has reference to them. i don't like it either